### ISSUE TITLE
Upload issues

### ISSUE THREAD

**Issue by slavaGanzin** [pressly/sup#114]
Hello thanks for your awesome project.

I had an issues with upload:

1. if ```src``` is not present I had no errors
1. ```-debug``` key didn't change this behaviour
1. there should be a message that something was(or not) copied
1. currently I see only tar (not so usefull) messages: ```tar: .config/fish/fundle/edc/bass/.git: time stamp 2017-06-20 15:47:40 is 199.705320196 s in the future ```
1. upload doesn't resolve local home dir using tilde ```~``` (As I found ```$HOME``` works, but...)
1. Relative path issue
```yaml
    upload:
      - src: ../.config/fish
        dst: /tmp

#will copy into /tmp/.config/fish
#but:
cp -r ../.config/fish /tmp
#will copy contents into /tmp/fish which is default behavior for unix
```

*****
```sh
sup -v                                                                  
# 0.5
uname -a
# Linux war 4.11.5-1-ARCH #1 SMP PREEMPT Wed Jun 14 16:19:27 CEST 2017 x86_64 GNU/Linux
go version
# go1.8.3 linux/amd64
```

Thanks

**Comment by VojtechVitek** [comment]
Hi, thanks for reporting this issues.

However, I won't really have time to debug and fix the cause in the near future. Any help or Pull Request would be greatly appreciated.

**Comment by slavaGanzin** [comment]
> (As I found ```$HOME``` works, but...)
no it's not :(

I'm far from golang. But I'll document this in readme

**Comment by slavaGanzin** [comment]
Before I was using:

```
local:
   scp -r from $SUP_HOST:to
```

for this task and the only problem I had I can get ```$SUP_HOST == 'localhost'``` in local section.
I think this was perfect solution and ```upload``` is ambiguous and unneeded :)

**Comment by RussellLuo** [comment]
There was also [an absolute path issue](https://github.com/pressly/sup/issues/109#issuecomment-297603100), but [the pull request](https://github.com/pressly/sup/pull/113) submitted to address this issue got no response :-(

**Comment by VojtechVitek** [comment]
Sorry, I've been very busy recently with other work and I can't focus on pressly/sup at this point.

My plan was to upgrade the whole `upload` thing with rsync and provide much better error messages.

**Comment by VojtechVitek** [comment]
#113 has been fixed, thanks @RussellLuo for the reminder!

**Comment by VojtechVitek** [comment]
This has been resolved in #113. Please reopen if I'm missing something.
